### ISSUE TITLE
Include more recommender info in recommender events

### ISSUE THREAD

**Issue by reckart** [inception-project/inception#770]
Events such as the `RecommenderEvaluationResultEvent` should include more information about the recommender, in particular after #763 which allows defining a custom recommender name. In particular that name should be included. It also wouldn't hurt to include the ID.

**Comment by reckart** [comment]
Including the ID in the log creates a new problem: when exporting a project (including the log) and importing that project again, the recommender IDs change and the imported log needs to be updated accordingly...

**Comment by reckart** [comment]
Opened a new issue for the ID problem: #816
